guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+| ---------------------------------------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| **Auto-start workspace (schedule)**                  | ✅                                                                    | ✅                                                                                                           |
+| **Auto-start workspace (SSH or visit app)**          | ❌                                                                    | 🤔 [#2909](https://github.com/coder/coder/issues/2909)                                                       |
+| **Code via web terminal**                            | ✅                                                                    | ✅                                                                                                           |
+| **Code via code-server (Code Web)**                  | ✅ Hardcoded version                                                  | ✅ Any version [via the template](https://coder.com/docs/coder-oss/latest/ides/web-ides#code-server)         |
+| **Code via JetBrains Projector (web)**               | ✅ Hardcoded version                                                  | ✅ Any version [via the template](https://coder.com/docs/coder-oss/latest/ides/web-ides#jetbrains-projector) |
+| **Code via SSH (VS Code Remote, JetBrains Gateway)** | ✅ With [coder-cli](https://github.com/coder/coder-cli) installed     | ✅ With [coder](https://coder.com/docs/coder-oss/latest/install) installed                                   |
+| **Custom workspace applications**                    | ✅                                                                    | ✅ Defined in [templates](https://coder.com/docs/coder-oss/latest/templates#coder-apps)                      |
+| **Access ports (SSH/tunnel)**                        | ✅                                                                    | ✅                                                                                                           |
+| **Access ports (web UI)**                            | ✅ [Dev URLs](https://coder.com/docs/coder/latest/workspaces/devurls) | ⌛ [(#1624)](https://github.com/coder/coder/issues/1624)                                                     |
+| **Share ports (web UI)**                             | ✅ [Dev URLs](https://coder.com/docs/coder/latest/workspaces/devurls) | ⌛ [(#3515)](https://github.com/coder/coder/issues/3515)                                                     |
+| **Docker in workspaces (Kubernetes)**                | ✅ [CVMs](https://coder.com/docs/coder/latest/workspaces/cvms)        | ⌛ [(needs docs)](https://github.com/coder/coder/issues/3376)                                                |
+
+<small>Something missing, or have feedback?
+[Let us know](https://coder.com/contact)</small>
+
+### Enterprise/management
```

Comment:
once again this chart comes out of no where. what is it's purpose. please give context. even if youre repeating yourself from previous explanation, there should be something new to add, but its better to repeat yourself a lil bit because the context is lost.